### PR TITLE
Dom massi edit additional information

### PR DESCRIPTION
**Pull request by terazus**

### CODE REVIEW

**Comment by terazus** [comment]
Dunno why the coverage report in the checks won't update but its here: https://coveralls.io/builds/38642374

**Comment by knirirr** [comment]
I noticed that the "save and continue" and "save and exit" buttons both act like the former on this tab.

File: tests/fixtures/additionalInformation.json [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            "default": "data access",
+            "description": "Data access is used when the process is accessing data, e.g. searching, browsing, viewing, and/or visualizing the data. Data curation is used when the process provides data curation tasks, e.g. submitting and/or editing the data. Data release is used when the process is releasing data, e.g. downloading and/or exporting the data. Data versioning is used when the process is versioning data, e.g. upload of new versions of data."
+          }
+        }
+      }
+    },
+    "description": "Properties specific to repositories",
+    "additionalProperties": false
+  },
+  "data": {
```
Comment (knirirr) [inline tests/fixtures/additionalInformation.json:297]: Is this something left over from debugging, or is it used somewhere?

**Review by knirirr — CHANGES_REQUESTED** [review]
This looks good to me, though there's one part of the field definitions I didn't understand (see comment) and the save button will need dealing with.

File: tests/fixtures/additionalInformation.json (hunk 1 above)
Comment (terazus) [reply tests/fixtures/additionalInformation.json:297]: This is a test fixture im using to run the schemas tests. What's the problem?

Comment (knirirr) [reply tests/fixtures/additionalInformation.json:297]: None. I couldn't spot where it was used and wondered about it.

**Comment by terazus** [comment]
Redirection should now behave correctly

**Review by knirirr — APPROVED** [review]

**Pull request merged**